tools/update-rust.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+set -e
+
+if [[ $# -lt 1 ]]; then
+    echo "Usage: $0 $RUST_VERSION"
+    echo
+    echo "Example: $0 1.71.0"
+    exit 2
+fi
+
+TOOLSDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ROOTDIR=$(realpath "${TOOLSDIR}/..")
+
+VERSION="${1}"
+OUTPUT="${ROOTDIR}/hashes/rust"
+PACKAGE_ROOT="https://static.rust-lang.org"
+METADATA_FILE="metadata.json"
+ARCHES=("x86_64" "aarch64")
+PACKAGES=("rust-std" "rustc" "cargo")
+
+function get_package_url {
+    local package="$1"
+    local arch="$2"
+
+    path=$(grep -e "${package}.*${arch}-unknown-linux-gnu.tar.xz" "${METADATA_FILE}" | cut -d '"' -f 2)
+    path=$(echo "${path//\\}")
```

Comment:
Why does the script need to strip backslashes? Nit: If it turns out to be needed, it could be just `path=${path//\\}`.

Reply:
I think originally I was pulling this a different way that was resulting in a trailing `\`. But that is no longer the case, and I've verified this line is not needed. Will clean that up.